src/main/java/com/google/sps/servlets/EventsServlet.java

```diff
@@ -81,18 +92,33 @@
     static final String START_TIME = "start";
     static final String END_TIME = "end";
     static final String USER_TIMEZONE = "timezone";
+    static final String EVENT_CREATOR = "creator";
+    static final String EVENT_NUM = "EventNum";
+    static final String EVENT_NUM_VALUE = "value";
+    static final String EVENT_ID = "event_id";
+    static final String ID_TOKEN_PARAM = "id_token";
+
+    private static final String CLIENT_ID = "605480199600-e4uo1livbvl58cup3qtd1miqas7vspcu.apps.googleusercontent.com";
 
     static final List<String> CATEGORIES = new ArrayList<String>(
         Arrays.asList("food_beverage", "nature", "water", "waste_cleanup", "other")
     );
 
     static final int MAX_STRING_BYTES = 1500;
 
+    static final HttpTransport HTTP_TRANSPORT = new UrlFetchTransport();
+    static final JsonFactory JSON_FACTORY = new GsonFactory();
+
+    static final GoogleIdTokenVerifier verifier = new GoogleIdTokenVerifier.Builder(HTTP_TRANSPORT, JSON_FACTORY)
+        .setAudience(Collections.singletonList(CLIENT_ID))
+        .build();
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Why move datastoreService out of each handler and up to here?

Reply:
Previously moved it so I wouldn't have to pass it to each member function that involved datastore, but most of that changed when I moved functionality into the User class so I can move it back to each handler.